src/translations/en.json

```diff
@@ -407,6 +407,42 @@
       }
     },
     "dialogs": {
+      "quick_open": {
```

Comment:
This still hasn't been moved

Reply:
Awaiting reply in the comment thread. Want clarification on something first